src/main/java/net/tridentsdk/meta/item/ItemDisplayProperties.java

```diff
@@ -54,6 +55,39 @@
      * @param lore Lore you wish to set it to
      */
     default void setLore(String... lore) {
-        setLore(Arrays.asList(lore));
+        this.setLore(Arrays.asList(lore));
+    }
+
+    /**
+     * Adds lore to the item starting from a specific index.
+     *
+     * @param index Index to begin insertion at. If equal to number of lore, it appends.
+     * @param lore Lore to be added.
+     *
+     * @see #addLore(String...)
+     */
+    default void addLore(int index, String... lore) {
+        List<String> curr = this.lore();
+        if (curr == null) {
+            curr = Lists.newArrayList();
+        }
+        if (curr.isEmpty() || index == curr.size()) {
```

Comment:
Remove equal to amount of lore feature. The small note on the javadoc may cause it to be hard to find if bugs appear.

Additionally, does addAll fit the functionality of this method? Would you like for the element at the given index to be shifted (as specified by the addAll method) or replaced (which you will need to do yourself)?